src/setuptools_scm/__init__.py

```diff
@@ -155,20 +163,21 @@ def _get_version(config: Configuration) -> str | None:
     parsed_version = _do_parse(config)
     if parsed_version is None:
         return None
-    version_string = _format_version(
+    version_str = _format_version(
         parsed_version,
         version_scheme=config.version_scheme,
         local_scheme=config.local_scheme,
     )
     if config.write_to is not None:
         dump_version(
             root=config.root,
-            version=version_string,
+            version=parsed_version,
```

Comment:
i recommend using the name scm_version or parsed_version as input names so it can be distinguished